tensorflow/tensorboard/BUILD

```diff
@@ -20,7 +20,10 @@ filegroup(
 
 py_binary(
     name = "tensorboard",
-    srcs = ["tensorboard.py"],
+    srcs = [
+        "tensorboard.py",
```

Comment:
I thinkyou need to sweap the two lines so  "__main__.py" comes first to prevent the buildifier error.